cint/cint.py

```diff
@@ -296,12 +319,65 @@ class U64(Cint, ctypes.c_uint64):
     SIZE = 8
 
 
+class F32(Cint, ctypes.c_float):
+    # MIN/MAX are not defined for floats
+    # Update: well, maybe they are definded after all
+    MIN = 2 ** -126
+    MAX = (2 - 2 ** -23) * 2 ** 127
+    UNSIGNED = False
+    CTYPEDEF = 'float'
+    SIZE = 4    
+
+    @classmethod
+    def from_le(cls, val):
```

Comment:
I think we may want to have such methods for any type. Also, this could be defined in a base class and just use `self.SIZE` as size